tests/test_scan.py

```diff
@@ -365,3 +374,119 @@ def test_scan_execute_with_soda_server_client_scan_result_does_not_contain_any_e
     )
 
     assert not scan_result.has_errors()
+
+
+def test_test_results_to_data_frame(spark_session: SparkSession) -> None:
+    """Test conversions of test_result to dataframe."""
+    expected = spark_session.createDataFrame(
+        [
+            {
+                "test": Test(
+                    id="id",
+                    title="title",
+                    expression="expression",
+                    metrics=["metrics"],
+                    column="column",
+                ),
+                "passed": True,
+                "skipped": False,
+                "values": {"value": "10"},
+                "error": "exception",
+                "group_values": {"group": "by"},
+            }
+        ]
+    )
+
+    test_results = [
+        TestResult(
+            Test(
+                id="id",
+                title="title",
+                expression="expression",
+                metrics=["metrics"],
+                column="column",
+            ),
+            passed=True,
+            skipped=False,
+            values={"value": 10},
+            error="exception",
+            group_values={"group": "by"},
+        )
+    ]
+    out = scan.testresults_to_data_frame(test_results)
+    assert (
+        expected.select(sorted(expected.columns)).collect()
+        == out.select(sorted(out.columns)).collect()
+    )
+
+
+def test_measurements_to_data_frame(spark_session: SparkSession) -> None:
+    """Test conversions of measurements to dataframe."""
+    expected = spark_session.createDataFrame(
+        [
+            {
+                "metric": "values_count",
+                "column_name": "officename",
+                "value": "",
+                "group_values": [
+                    GroupValue(group={"statename": "statename"}, value="9872")
+                ],
+            }
+        ]
+    ).withColumn(
+        "value", when(col("value") == "", None).otherwise(col("value"))
```

Comment:
can't this be solved by changing line 430 to: `"value": None`?

Reply:
@JCZuurmond The schema define `value` as StringType(). In the json if we make it `"value": None` , the dataframe fails. I am leaving this as is for now.